config/fedora.yml

```diff
@@ -1,15 +1,15 @@
 development:
   user: fedoraAdmin
   password: fedoraAdmin
-  url: http://127.0.0.1:<%= ENV['FCREPO_DEVELOPMENT_PORT'] || 8984 %>/rest
+  url: <%= ENV['FCREPO_DEVELOPMENT_URL'] || "http://127.0.0.1:8984/rest" %>
```

Comment:
Use single quotes when not needing string interpolation. Make sure you run rubocop before pushing. TravisCI will fail on you for this.